evalml/pipelines/components/transformers/preprocessing/time_series.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+
+from evalml.pipelines.components.transformers.transformer import Transformer
+
+
+class DelayedFeaturesTransformer(Transformer):
```

Comment:
Also could you please add class and init docstrings?

Reply:
Will change the name! I thought plural was appropriate because it will lag more than one feature if there are multiple input features.

Reply:
Thanks. Yep got it, that makes sense, I'm fine either way but prefer the singular here because I think its more in line with our other components. I also recognize its a nit-pick / opinion haha